### PR TITLE
OSM Login Auth API 

### LINKED ISSUE #33: OSM LOGIN API Design
- Extends #13  
Plan is to create necessary endpoints for OSM Login API , Integrating https://github.com/kshitijrajsharma/osm-login-python to the project ! I want to pick this issue to go !

### PR DESCRIPTION
**Pull request by kshitijrajsharma**
- Resolve #33

### CODE REVIEW

**Comment by kshitijrajsharma** [comment]
I will mark the PR as ready for review when its ready , Will use it to update the progress  
cc : @krtonga , @robsavoye  , @ivangayton

File: src/backend/db/database.py [hunk 1]
```diff
@@ -16,12 +16,12 @@
...
 #
 
-from sqlalchemy import create_engine, MetaData
+from sqlalchemy import MetaData, create_engine
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
 
 # TODO don't hardcode this
-SQLALCHEMY_DATABASE_URL = 'postgresql+psycopg2://fmtm:fmtm@db:5432'
+SQLALCHEMY_DATABASE_URL = "postgresql+psycopg2://admin:admin@localhost:5432/postgres"
```
Comment (krtonga) [inline src/backend/db/database.py:24]: Is this meant to be included?

Comment (kshitijrajsharma) [reply src/backend/db/database.py:24]: No , should be read from .env , I will include its url

Comment (robsavoye) [reply src/backend/db/database.py]: So you'll push the PR again after that fix to read the .env file ?

Comment (kshitijrajsharma) [reply src/backend/db/database.py]: yes , converting pr to draft to work on it

**Comment by kshitijrajsharma** [comment]
It will break docker image if we have any , API should be running from it's  root dir , just like frontend

**Pull request merged**